src/test/java/registrationTest.java

```diff
@@ -0,0 +1,36 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class registrationTest extends BaseTest {
+
+    @Test
+    public static void registrationNavigation() {
+
+        WebDriver driver = new ChromeDriver();
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(5));
+
+        // go to url
+        String url = "https://bbb.testpro.io/";
+        driver.get(url);
+
+        // find the registration link
+        WebElement registrationlink = driver.findElement(By.linkText("Registration"));
+        registrationlink.click();
+
+        //verify the page title of the registration page
+        String expectedTitle = "Koel Registration page";
+        String actualTitle = driver.getTitle();
+        Assert.assertEquals(actualTitle, expectedTitle);
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._